client/blocks/reader-site-subscription/site-subscription-subheader.tsx

```diff
@@ -67,20 +66,19 @@ const SiteSubscriptionSubheader = ( {
 	const recordViewFeedButtonClicked = useRecordViewFeedButtonClicked();
 
 	subheaderItems.push(
-		<Button
-			key={ `view-feed-button-${ feedId }` }
-			title={ translate( 'View feed' ) }
-			className="site-subscription-header__view-feed-button"
-			icon={ <FeedIcon /> }
+		<a
 			href={ getFeedUrl( feedId ) }
+			title={ translate( 'View feed' ) }
 			onClick={ () => {
 				recordViewFeedButtonClicked( {
 					blogId: blogId ? String( blogId ) : null,
 					feedId: String( feedId ),
-					source: 'subscription-feed-icon',
+					source: 'subscription-feed-link',
 				} );
 			} }
-		/>
+		>
+			Reader
```

Comment:
It's a product name but elsewhere we seem to wrap Reader in translation strings, e.g.:

https://github.com/Automattic/wp-calypso/blob/92c289eeb28b00630c94316f45643c10bf16a31f/client/reader/sidebar/index.jsx#L298

https://github.com/Automattic/wp-calypso/blob/92c289eeb28b00630c94316f45643c10bf16a31f/client/layout/masterbar/logged-in.jsx#L597

Reply:
fixed 👍